backend/src/core/security.py

```diff
@@ -9,13 +10,21 @@
 __crypt_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
 
 
-def create_access_token(subject: int) -> str:
-    expires_at = datetime.now(timezone.utc) + timedelta(minutes=settings.ACCESS_TOKEN_EXPIRE_MINUTES)
+def create_access_token(subject: int | str, expire=settings.ACCESS_TOKEN_EXPIRE_MINUTES) -> str:
+    expires_at = datetime.now(timezone.utc) + timedelta(minutes=expire)
     to_encode = JWTPayload(exp=expires_at, sub=subject)
     encoded_jwt = jwt.encode(to_encode.model_dump(), settings.SECRET_KEY, algorithm=settings.ALGORITHM)
     return encoded_jwt
 
 
+def decode_access_token(token: str):
+    try:
+        decoded_token = jwt.decode(token, settings.SECRET_KEY, algorithms=settings.ALGORITHM)
+        return str(decoded_token["sub"])
```

Comment:
and after changing above do this:
```suggestion
        return data.sub
```